examples/seismic/skew_self_adjoint/example_iso.py

```diff
@@ -55,9 +55,9 @@ def run(shape=(50, 50, 50), spacing=(10.0, 10.0, 10.0), tn=1000.0,
     info("Applying Adjoint")
     solver.adjoint(rec, autotune=autotune)
     info("Applying Born")
-    solver.born(dm, autotune=autotune)
+    solver.jacobain(dm, autotune=autotune)
```

Comment:
jacobIAn typos?

Reply:
Are thos examples being tested?

Reply:
indeed, good point. I'd say let's add a tiny test to test_benchmark.py

Reply:
Good catch. 
>. I'd say let's add a tiny test to test_benchmark.py
Already there with correct name, this is in the example

Reply:
fixed